examples/basics/square-moving.cpp

```diff
@@ -16,13 +19,9 @@ using namespace std;
 
 // keys to InputHandler observe
 std::map<const char*, SDL_Keycode> input_config = {
-    {"quit", SDLK_ESCAPE},
-    {"up", SDLK_UP},
-    {"left", SDLK_LEFT},
-    {"right", SDLK_RIGHT},
-    {"down", SDLK_DOWN},
-    {"velocity_up", SDLK_w},
-    {"velocity_down", SDLK_s},
+    { "quit", SDLK_ESCAPE },     { "up", SDLK_UP },     { "left", SDLK_LEFT },
```

Comment:
I can't disable this if I enable this option: https://github.com/NewayPix/hazard-memory/pull/45/commits/fccfeb0ad7fbb2641a99b7cd9a1da35be411ef51